### DISCUSSION TITLE
Crash while route-calculation

### DISCUSSION THREAD

**Discussion by ThomasMeister123** [osmandapp/OsmAnd#15834] [Q&A] [answered]
Osmand crashes on EVERY route-calculation >100 km (circa).
Progress bar goes slowly up to arround 80%. Then it stops. After ca 15-30 seconds  Osmand closes without warning.

Maps are all load (route within Germany) 

It crashes with different destinations and different routing-profiles (also with the original car-profile)

Has somebody an idea? How can I discover what the problem is?

**Comment by vshcherb** [comment, marked as answer]
There was issue with first generation of Germany maps, please delete them all and download again. It should work fine

**Reply by ThomasMeister123** [reply]
Indeed. After I downloaded the german maps again, it works.
When there is such a bug in the map-data it would be nice to mark the fixed data as "new map", so that it will be shown in Updates
